piserver/app.py

```diff
@@ -22,15 +23,14 @@
 # INIT BLINDS SYSTEM RELATED COMPONENTS #
 temp_sensor = BME280TemperatureSensor() if app.config["USE_TEMP_SENSOR"] \
         else MockTemperatureSensor()
-smart_blinds_system =  SmartBlindsSystem( Blinds( None ), None, temp_sensor )
+#TODO: Init a motor hanndler here
 
-@app.route('/')
-def index():
-    return 'Server Works!'
-  
-@app.route('/greet')
-def say_hello():
-    return 'Hello from Server'
+# default empty schedule 
```

Comment:
I think you should've kept the old routes that just return a value (at least one of them). They could be useful when trying to debug connectivity issues with the server/rpi.

Reply:
But idk, what do you think?

Reply:
Fair enough. I can put them back.

Reply:
Done